plugins/birdnet-go/manager.py

```
@@ -0,0 +1,1079 @@
+                self.mqtt_client = mqtt.Client(client_id=self.mqtt_client_id, clean_session=True)
+
+            self.mqtt_client.on_connect = self._on_mqtt_connect
+            self.mqtt_client.on_disconnect = self._on_mqtt_disconnect
+            self.mqtt_client.on_message = self._on_mqtt_message
+
+            if self.mqtt_username:
+                self.mqtt_client.username_pw_set(self.mqtt_username, self.mqtt_password)
+
+            self.mqtt_client.connect(self.mqtt_host, self.mqtt_port, self.mqtt_keepalive)
+            return True
+        except Exception as e:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)